modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/restclients/OrgMgtRestClient.java

```diff
@@ -0,0 +1,394 @@
+
+        selfServiceAppClientId = openIDConnectConfiguration.getClientId();
+        selfServiceAppClientSecret = openIDConnectConfiguration.getClientSecret();
+        ClientID clientID = new ClientID(selfServiceAppClientId);
+        Secret clientSecret = new Secret(selfServiceAppClientSecret);
+        ClientAuthentication clientAuth = new ClientSecretBasic(clientID, clientSecret);
+
+        AuthorizationGrant clientCredentialsGrant = new ClientCredentialsGrant();
+        Scope scope = new Scope("SYSTEM");
+
+        return new TokenRequest(tokenEndpoint, clientAuth, clientCredentialsGrant, scope);
+    }
+
+    private String getTenantQualifiedURL(String endpointURL, String tenantDomain) {
+
+        if (!tenantDomain.isBlank() && !tenantDomain.equalsIgnoreCase(
+                MultitenantConstants.SUPER_TENANT_DOMAIN_NAME)) {
+
+            endpointURL = endpointURL.replace(baseUrl,
+                    baseUrl + TENANT_PATH + tenantDomain + PATH_SEPARATOR);
+        }
+        return endpointURL;
+    }
+
+    private Header[] getHeaders() {
+
+        Header[] headerList = new Header[3];
+        headerList[0] = new BasicHeader(USER_AGENT_ATTRIBUTE, OAuth2Constant.USER_AGENT);
+        headerList[1] = new BasicHeader(AUTHORIZATION_ATTRIBUTE, BASIC_AUTHORIZATION_ATTRIBUTE +
+                Base64.encodeBase64String((authenticatingUserName + ":" + authenticatingCredential).getBytes()).trim());
+        headerList[2] = new BasicHeader(CONTENT_TYPE_ATTRIBUTE, String.valueOf(ContentType.JSON));
+
+        return headerList;
+    }
+
+    private Header[] getHeadersWithBearerToken(String accessToken) {
+
+        Header[] headerList = new Header[3];
+        headerList[0] = new BasicHeader(USER_AGENT_ATTRIBUTE, OAuth2Constant.USER_AGENT);
+        headerList[1] = new BasicHeader(AUTHORIZATION_ATTRIBUTE, BEARER_TOKEN_AUTHORIZATION_ATTRIBUTE +
+                accessToken);
+        headerList[2] = new BasicHeader(CONTENT_TYPE_ATTRIBUTE, String.valueOf(ContentType.JSON));
+
+        return headerList;
+    }
+
+    private String buildPath(String serverUrl, String tenantDomain, String endpointURL) {
+
+        if (tenantDomain.equals(MultitenantConstants.SUPER_TENANT_DOMAIN_NAME)) {
+            return serverUrl + API_SERVER_BASE_PATH + endpointURL;
+        } else {
+            return serverUrl + TENANT_PATH + tenantDomain + PATH_SEPARATOR + API_SERVER_BASE_PATH + endpointURL;
+        }
+    }
+
+    private String getSubOrgOrganizationsPath(String serverUrl, String tenantDomain) {
+
+        if (tenantDomain.equals(MultitenantConstants.SUPER_TENANT_DOMAIN_NAME)) {
+            return serverUrl + ORGANIZATION_PATH + API_SERVER_BASE_PATH + ORGANIZATION_MANAGEMENT_PATH;
+        } else {
+            return serverUrl + TENANT_PATH + tenantDomain + PATH_SEPARATOR + ORGANIZATION_PATH + API_SERVER_BASE_PATH +
+                    ORGANIZATION_MANAGEMENT_PATH;
+        }
```

Comment:
```suggestion
        }
        return serverUrl + TENANT_PATH + tenantDomain + PATH_SEPARATOR + ORGANIZATION_PATH + API_SERVER_BASE_PATH +
                    ORGANIZATION_MANAGEMENT_PATH;
```

Reply:
Resolved with https://github.com/wso2/product-is/pull/20772/commits/1258e75f5c9e9525d851fd5e4143513bfc72946c